components/sections/transactionsection.tsx

```diff
@@ -731,9 +742,9 @@ export default function TransactionSection() {
             <ArrowLeft />
           </button>
           <button
-            disabled={currentPage === paginatedTransactionArray.length - 1}
+            disabled={pendingTab ? currentPage === paginatedTransactionArray.length - 1 : currentPage === paginatedCompletedTransactionArray.length - 1}
             onClick={() => setCurrentPage((prev) => prev + 1)}
-            className={`rounded-lg bg-[#484C5D] ${currentPage === paginatedTransactionArray.length - 1
+            className={`rounded-lg bg-[#484C5D] ${pendingTab ? currentPage === paginatedTransactionArray.length - 1 : currentPage === paginatedCompletedTransactionArray.length - 1
```

Comment:
Those could be variable above for better clarity

Reply:
fixing this